eng/pipelines/windows.yml

```diff
@@ -96,12 +96,6 @@ jobs:
               _framework: uap
               _helixQueues: $(uapNetfxQueues)
 
-            UAP_arm_Release:
```

Comment:
Can we at least keep building UAP without running tests? That will protect the build.

Reply:
I'm not sure how to do that? Remove only _helixQueues ???

Reply:
Nope, move it down to the Job called WindowsNoTest under the if statement IsOfficialBuild == true.

Reply:
done.